main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -705,8 +705,8 @@ object Resolver {
     case NamedAst.Declaration.AssocTypeDef(doc, mod, ident, arg0, tpe0, loc) =>
 
       // For now, we don't add any tvars from the args. We should have gotten those directly from the instance
-      val argVal = resolveType(arg0, Wildness.ForbidWild, env, taenv, ns0, root)
-      val tpeVal = resolveType(tpe0, Wildness.ForbidWild, env, taenv, ns0, root)
+      val argVal = resolveType(arg0, None, Wildness.ForbidWild, env, taenv, ns0, root)
+      val tpeVal = resolveType(tpe0, None, Wildness.ForbidWild, env, taenv, ns0, root)
       val symVal = trt.assocs.collectFirst {
```

Comment:
@mlutze Can we predict the kind of either of the above types? Probably not, right?